From 2fadadf1075c95b2955f047fa0387b39612f7b30 Mon Sep 17 00:00:00 2001
From: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
Date: Thu, 30 May 2024 13:37:38 +0000
Subject: [PATCH] plat-corstone1000: Remove MMCOMM buffer address

Remove MMCOMM buffer address and mapping, as it is not being used anymore

Upstream-Status: Backport [In v3, https://github.com/OP-TEE/optee_os/commit/eaee88fbcac6dcc15fe1d1a758b53eb2b66cfc60]
Signed-off-by: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
---
 core/arch/arm/plat-corstone1000/main.c            | 1 -
 core/arch/arm/plat-corstone1000/platform_config.h | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/core/arch/arm/plat-corstone1000/main.c b/core/arch/arm/plat-corstone1000/main.c
index b3861c4c8..61171b9a9 100644
--- a/core/arch/arm/plat-corstone1000/main.c
+++ b/core/arch/arm/plat-corstone1000/main.c
@@ -15,7 +15,6 @@
 static struct pl011_data console_data __nex_bss;
 
 register_ddr(DRAM0_BASE, DRAM0_SIZE);
-register_ddr(MM_COMM_BUF_BASE, MM_COMM_BUF_SIZE);
 
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, CONSOLE_UART_BASE, PL011_REG_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, GICD_BASE, GIC_DIST_REG_SIZE);
diff --git a/core/arch/arm/plat-corstone1000/platform_config.h b/core/arch/arm/plat-corstone1000/platform_config.h
index 6653d38bc..f59c93a14 100644
--- a/core/arch/arm/plat-corstone1000/platform_config.h
+++ b/core/arch/arm/plat-corstone1000/platform_config.h
@@ -26,9 +26,6 @@
 #define GICD_BASE		(GIC_BASE + GICD_OFFSET)
 #define GICC_BASE		(GIC_BASE + GICC_OFFSET)
 
-#define MM_COMM_BUF_BASE	0x02000000
-#define MM_COMM_BUF_SIZE	0x1000
-
 #define UART_BAUDRATE		115200
 #define CONSOLE_BAUDRATE	UART_BAUDRATE
 
-- 
2.34.1

